pkgs/development/libraries/libminc/default.nix

```diff
@@ -12,11 +12,10 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ cmake ];
   buildInputs = [ zlib netcdf hdf5 ];
 
-  cmakeFlags = [ "-DBUILD_TESTING=${if doCheck then "ON" else "OFF"}"
+  cmakeFlags = [ "-DLIBMINC_BUILD_SHARED_LIBS=ON"
                  "-DLIBMINC_MINC1_SUPPORT=ON" ];
 
   checkPhase = "ctest";
-  doCheck = true;
```

Comment:
Note that `doCheck` defaults to false, so you're effectively disabling checking here.